backend/config/locales/zu.yml

```diff
@@ -1,4 +1,19 @@
 zu:
+  backoffice:
+    backoffice: Backoffice
+    welcome: Welcome to HeCO Invest backoffice.
+    enter_details_below: Please enter your details below.
+    sign_out: Sign out
+    sign_in: Sign in
+    name: Name
+    account_owner: Account owner
+    account_users: Account users
+    projects: Projects
+    language: Language
+    status: Status
+    approve: Approve
+    reject: Reject
+    project_developers: Project developers
```

Comment:
I think it would be worthy to split translations to subcategories (per tables for example) so we don't have everything under one key (`backoffice `)

Reply:
Yeah, I was thinking about it, but then you have multiple same strings to translate. Dunno, I'm kinda like the frontend style of doing this, generate random id for unique strings, but they have tools that does that. Can you give me example how would you split it? Per view? Common actions separate?

Reply:
Hmm, that is always difficult to answer and it happens quite often in Rails that there are duplicities at locales, but I would for example go for `general` section which covers these string which are used everywhere and then add key per all other sections --> `project_developers`, `investors`, etc.

Reply:
Like this? Kinda rails style so lazy lookup for some pages can be used
```
common:
      name: Name
      status: Status
      language: Language
      approve: Approve
      reject: Reject
    sign_in:
      welcome: Welcome to HeCO Invest backoffice.
      enter_details_below: Please enter your details below.
      sign_in: Sign in
    project_developers:
      index:
        name: Name
        account_owner: Account owner
        account_users: Account users
        projects: Projects
        language: Language
        status: :backoffice.common.status
        approve: :backoffice.common.approve
        reject: :backoffice.common.reject
    layout:
      header: Backoffice
      sign_out: Sign out
      project_developers: Project developers
```

Reply:
yp, looks better for sure :), I normally dont use `:backoffice.common.status` and just take it right from there --> `I18n.t("backoffice.common.status")`, but if this works with transifex, no problem with that

Reply:
Hmm, I had in this way in different project and translator did not mind, as I see it creates extra strings that should not be changed. Maybe as you said use common, but then the whole point of having things separately falls apart for me haha.

Reply:
:D I think this is more about to keep some kind of structure in locales so you don't have 100 words under one key, but it does not have to be so rigid so you can use at some cases translations from different sections <-- but agree that it is not optimal :)

Reply:
Ok, let's experiment a little with this. I think we have some time to refactor the structure until translations start. To be honest, I have nothing against having all keys under just `backoffice` section, as not have to think about the structure too much is also a virtue. Or we can just always duplicate stuff.